release/ray_release/schema.json

```diff
@@ -0,0 +1,174 @@
+{
```

Comment:
any chance we can define this in pydantic instead? So we don't need to hand write the json schema as the types evolves.